### ISSUE TITLE
A11y_Dynamics for Microsoft_My Labor Entries_Keyboard:  Unable to navigate to table headers using keyboard.

### ISSUE THREAD

**Issue by Accessibility56** [OfficeDev/office-ui-fabric-react#10366]
[Check out Accessibility Insights! ](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx)- Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
https://cs-labor-test.crm.dynamics.com/main.aspx
Server version: 9.1.0000.6228
Client version: 1.3.1948-190626-193951

**Browser Details: ​**
OS: Windows (Version 1903, Build: 18362.239)
Microsoft Edge 44.17763.1.0, Chrome Version 75.0.3770.142 (Official Build) (64-bit)

**Repro Steps: ​**
1)Launch the Url: https://cs-labor-test.crm.dynamics.com/main.aspx and login with appropriate Credentials
2)Tab to all controls and verify whether user is able to navigate to table header headers using keyboard.
 
**Actual Result:** ​
Unable to navigate to table headers using keyboard.

**Expected Result:** ​
User should be able to navigate to table headers using keyboard.

**User Impact: ​**
Keyboard only users can not able to perform all operations if few controls are not navigable using keyboard.

**MAS Reference: ​**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
[Bug_10366_Keyboard issue.pptx](https://github.com/OfficeDev/office-ui-fabric-react/files/3577549/Bug_10366_Keyboard.issue.pptx)

**Comment by Accessibility56** [comment]
Copied from https://microsoft.visualstudio.com/OSGS/_workitems/edit/23027275

**Comment by KevinTCoughlin** [comment]
@Accessibility56 what version of the office-ui-fabric-react library is in use on the page showcasing the above behavior?

**Comment by Accessibility56** [comment]
@KevinTCoughlin   The office fabric version is office-ui-fabric-react": "^6.9.0"

**Comment by dzearing** [comment]
Can't access the test page. No details in this bug on repro steps. If you are giving no effort in deducing an accessibility bug for your app, we can't help. As this is right now, there is no action we can take.

If you deduce that you are using a Fabric component and that it has a missing behavior causing keyboard accessibility issues, please be clear about the repo. Which component, what were the props passed in, have you repro'd it in isolation?

If you put in that effort and have a clear repro and expected/resulted listed for us, such that we can take action, we're glad to help.
